suites/portal/fileExplorerTest.js

```diff
@@ -0,0 +1,56 @@
+Feature('File Explorer');
```

Comment:
Looks like other test scripts have a reference to the corresponding JIRA ID that tracks this task, maybe we should follow this standard:
e.g.,
> "Feature('<some description> - https://ctds-planx.atlassian.net/browse/<JIRA ID>');"

Reply:
@themarcelor, will do